policies/technical/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: OpenSSL Technical Policies
+subtitle: |
+    First issued 30th September 2020
+    <br />
```

Comment:
Isn't this just `<br>`?

Reply:
I still have XHTML in my fingers :wink:

This is permitted in HTML 5, though...

If you want to enjoy a discussion on the topic, I recommend
https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br

Reply:
BTW, not my fault...  it's already there in the file this came from:

https://github.com/openssl/web/blob/68f71e63270307dde5343cfc5b0f31b49cbd3167/policies/otc-policies.html#L15-L16

Reply:
You added a space, so it's obviously your fault :)